### PR TITLE
Added HA config for radiator valve Siterwell GS361

### PR DESCRIPTION
**Pull request by dzegarra**
This PR depends on:

- [Added support for radiator valve Siterwell GS361 - zigbee-herdsman-converters](https://github.com/Koenkk/zigbee-herdsman-converters/pull/1120)
- [Handle the command 0x02 of the cluster manuSpecificTuyaDimmer - zigbee-herdsman](https://github.com/Koenkk/zigbee-herdsman/pull/162)

### CODE REVIEW

File: lib/extension/homeassistant.js [hunk 1]
```diff
@@ -468,6 +468,48 @@ const cfg = {
...
     },
 
+    'child_lock': {
+        type: 'lock',
+        object_id: 'child_lock',
+        discovery_payload: {
+            state_topic: true,
+            command_topic: true,
+            command_topic_postfix: 'child_lock',
+            payload_lock: '1',
```
Comment (Koenkk) [inline lib/extension/homeassistant.js:478]: Please change 1/0 to LOCK, UNLOCK

File: lib/extension/homeassistant.js [hunk 2]
```diff
@@ -468,6 +468,48 @@ const cfg = {
...
+
+    'window_detection': {
+        type: 'switch',
+        object_id: 'window_detection',
+        discovery_payload: {
+            state_topic: true,
+            command_topic: true,
+            command_topic_postfix: 'window_detection',
+            payload_off: '0',
+            payload_on: '1',
```
Comment (Koenkk) [inline lib/extension/homeassistant.js:494]: Please change 1/0 to ON/OFF

File: lib/extension/homeassistant.js [hunk 3]
```diff
@@ -468,6 +468,48 @@ const cfg = {
...
+
+    'valve_detection': {
+        type: 'switch',
+        object_id: 'valve_detection',
+        discovery_payload: {
+            state_topic: true,
+            command_topic: true,
+            command_topic_postfix: 'valve_detection',
+            payload_off: '0',
+            payload_on: '1',
```
Comment (Koenkk) [inline lib/extension/homeassistant.js:508]: Please change 1/0 to ON/OFF

File: lib/extension/homeassistant.js (hunk 1 above)
Comment (dzegarra) [reply lib/extension/homeassistant.js]: done

File: lib/extension/homeassistant.js (hunk 3 above)
Comment (dzegarra) [reply lib/extension/homeassistant.js]: done

**Comment by dzegarra** [comment]
I also updated the values on the [other PR](https://github.com/Koenkk/zigbee-herdsman-converters/pull/1120/commits/924be9f7b0e57cfbc734eed374c99a8db00bb584) to match your suggested changes.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**